nasa/earthdata-search
# EDSC-4238: Added Highlighting to GranuleResultsItem and converted to RTL from Enzyme testing

# Overview

### What is the feature?

Added a component that Highlights a searched granuleId.

### What is the Solution?

Added a new library with a component that highlights text.

### What areas of the application does this impact?

List impacted areas.

# Testing

### Reproduction steps

- **Environment for testing:**
- **Collection to test with:**

1. go to localhost:8080/search
2. Click on a collection
3. put in *(some value in the granule names)*
4. click enter.
5. observe the highlighted granule names

### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist
